src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1992,4 +2011,90 @@ export class PatientViewPageStore {
     @computed get genomeNexusInternalClient() {
         return new GenomeNexusAPIInternal(this.referenceGenomeBuild);
     }
+
+    @computed get sampleMsiHInfo() {
+        return getSampleClinicalDataMapByThreshold(
+            this.clinicalDataForSamples.result,
+            CLINICAL_ATTRIBUTE_ID_ENUM.MSI_SCORE,
+            MSI_H_THRESHOLD
+        );
+    }
+
+    @computed get sampleTmbHInfo() {
+        return getSampleClinicalDataMapByThreshold(
+            this.clinicalDataForSamples.result,
+            CLINICAL_ATTRIBUTE_ID_ENUM.TMB_SCORE,
+            TMB_H_THRESHOLD
+        );
+    }
+
+    getOtherBiomarkerQueries(
+        type: OtherBiomarkersQueryType,
+        sampleClinicalData: { [key: string]: ClinicalData }
+    ) {
+        return _.values(sampleClinicalData).map(clinical => {
+            return {
+                id: getOtherBiomarkersQueryId({
+                    sampleId: clinical.sampleId,
+                    type: type,
+                }),
+                alteration: OTHER_BIOMARKER_NAME[type],
+                gene: {
+                    hugoSymbol: OTHER_BIOMARKER_HUGO_SYMBOL,
+                },
+                tumorType: tumorTypeResolver(
+                    getSampleTumorTypeMap(
+                        this.clinicalDataForSamples.result,
+                        this.studyMetaData.result?.cancerType.name
+                    )
+                ),
+            } as AnnotateMutationByProteinChangeQuery;
+        });
+    }
+
+    @computed get otherBiomarkerQueries() {
+        const queries: AnnotateMutationByProteinChangeQuery[] = [];
+        if (_.keys(this.sampleMsiHInfo).length > 0) {
+            queries.push(
+                ...this.getOtherBiomarkerQueries(
+                    OtherBiomarkersQueryType.MSIH,
+                    this.sampleMsiHInfo
+                )
+            );
+        }
+        if (_.keys(this.sampleTmbHInfo).length > 0) {
+            queries.push(
+                ...this.getOtherBiomarkerQueries(
+                    OtherBiomarkersQueryType.TMBH,
+                    this.sampleTmbHInfo
+                )
+            );
+        }
+        return queries;
+    }
+
+    readonly getOtherBiomarkersOncoKbData = remoteData<{
+        [sampleId: string]: { [queryType: string]: IndicatorQueryResp };
+    }>({
+        invoke: async () => {
+            const allResult = await oncokbClient.annotateMutationsByProteinChangePostUsingPOST_1(
```

Comment:
POST_1?

Reply:
Yeah, the oncokb has two sets of apis and they sharing the same name. The swagger generates as POST_1.